helm-branches.yaml

```diff
@@ -1,35 +1,54 @@
-include:
-  - remote: 'https://raw.githubusercontent.com/jobtome-labs/ci-templates/v2.29.1/docker.yml'
-  - remote: 'https://raw.githubusercontent.com/jobtome-labs/ci-templates/v2.29.1/templates/helm.yml'
```

Comment:
don't you need at least `templates.helm`?

Reply:
I don't, because this file has to be included as an addition to the helm-regional or helm-multiregion include. So including templates.helm would lead to duplications here.

Reply:
but in the `deploy:quality:helm:branches` you `extend` `.helm:deploy` which is not in the same file (same for `.stop`)

where are they?

Reply:
ah nvm, i forgot that helm-branch is alongside helm-quality, and not a replacement